### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by japgolly**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+  p => p.settings(
+    scalacOptions ++= {
+      val isDotty = scalaVersion.value startsWith "3"
+      val ver     = version.value
+      if (isSnapshot.value)
+        Nil
+      else {
+        val a = p.base.toURI.toString.replaceFirst("[^/]+/?$", "")
+        val g = s"https://raw.githubusercontent.com/scala-js/scala-js-dom"
+        val flag = if (isDotty) "-scalajs-mapSourceURI" else "-P:scalajs:mapSourceURI"
```
Comment (japgolly) [inline build.sbt:13]: Note the different flag for Scala 3

**Comment by armanbilge** [comment]
Maybe you should target 1.x with this?

**Comment by japgolly** [comment]
Nah cos Scala 3 isn't on 1.x. I had the same confusion lol

**Comment by armanbilge** [comment]
I agree, so we'll add that in when we merge 1.x into 2.x :) btw this is how http4s/http4s manages three series simultaneously

**Pull request closed without merging**